cumulus/parachains/runtimes/constants/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
```

Comment:
License is probably not the right one. It would either be the [default one](https://github.com/paritytech/polkadot-sdk/blob/e5bb11b008ca6c1b5a07cab94a2327538f3bf15c/cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs#L2) or the [Cumulus one](https://github.com/paritytech/polkadot-sdk/blob/e5bb11b008ca6c1b5a07cab94a2327538f3bf15c/cumulus/test/runtime/src/lib.rs#L2).

Reply:
thank you, fixed here: https://github.com/paritytech/polkadot-sdk/pull/3134/commits/421fd96fc3f4d78bf443edf1e4bd9c6c37909ba3